public/js/components/AtomEdit/CustomEditors/CTAEditor.js

```diff
@@ -18,19 +18,19 @@ export class CTAEditor extends React.Component {
     return (
       <div className="editor editor-cta">
         <ManagedForm data={this.props.atom} updateData={this.props.onUpdate}>
-          <ManagedField fieldLocation="data.url" name="Link Url" isRequired={true}>
+          <ManagedField fieldLocation="data.url" name="Link Url" fieldId="url" isRequired={true}>
```

Comment:
that'd be fine. I'm not actually using it atm but thought a UID would be useful if we had a list of all field errors.